druntime/src/core/thread/osthread.d

```diff
@@ -186,6 +186,16 @@ else
 // Thread
 ///////////////////////////////////////////////////////////////////////////////
 
+private Thread toThread(return scope ThreadBase t) @trusted nothrow @nogc pure
+{
+    return cast(Thread) cast(void*) t;
+}
+
+private extern(D) static void thread_yield() @nogc nothrow
```

Comment:
This is private and not used anywhere in this module. Presumably I should remove it?